pkg/skaffold/schema/latest/config.go

```diff
@@ -1125,6 +1125,9 @@ type DockerArtifact struct {
 	// Secret contains information about a local secret passed to `docker build`,
 	// along with optional destination information.
 	Secret *DockerSecret `yaml:"secret,omitempty"`
+
+	// SSH is passed as a argument to the --ssh option.
+	SSH string `yaml:"ssh,omitempty"`
```

Comment:
I was wondering what to do here. "--ssh" option can have only one argument "default", "<id> [= <socket>]" or "<key> [, <key>]".
It would be nice if we could used a type representation like a union type, but the JSON schema(and Golang) doesn't support such a type, so I decided to just make it string type.

Reply:
SGTM — I think we could use a yaml `oneOf` annotation, but this will do for now.